sql-plugin/src/main/scala/org/apache/spark/sql/rapids/RapidsShuffleInternalManagerBase.scala

```diff
@@ -679,7 +675,6 @@ abstract class RapidsShuffleThreadedReaderBase[K, C](
             }
             val res = currentIter.next()
             val fetchTime = System.nanoTime() - fetchTimeStart
-            deserializationTimeNs.foreach(_ += (fetchTime - readBlockedTime))
```

Comment:
I'm not sure the new deserialization time metric does the same thing the old metric did.  The new metric is timing how long it takes the serializer to produce a new batch, but that's measuring I/O time as well as deserialization time once the I/O is completed.  The original metric appears to go out of its way to remove the time spent on I/O to get a more accurate deserialization metric.